io/disk/tiobench.py

```diff
@@ -63,22 +73,66 @@ def setUp(self):
         archive.extract(tarball, self.teststmpdir)
         os.chdir(os.path.join(self.teststmpdir, "tiobench-master"))
         build.make(".")
-        self.target = self.params.get('dir', default=self.workdir)
         self.disk = self.params.get('disk', default=None)
-
+        self.target = self.workdir
```

Comment:
@smruti77 Can we initialise this as `self.target = self.disk` ?
Reason is: When no_fs no_lv no_raid, we need the test to run on disk. But now it runs on workdir.

Reply:
Same for all tests.

Reply:
ok